app/Http/Controllers/MessagesController.php

```diff
@@ -14,7 +14,7 @@
 
 class MessagesController extends Controller
 {
-    public function index(){
+    public function index() {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->